homeassistant/components/vesync/coordinator.py

```diff
@@ -7,24 +7,32 @@
 
 from pyvesync import VeSync
 
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
 
 from .const import UPDATE_INTERVAL
 
 _LOGGER = logging.getLogger(__name__)
 
+type VeSyncConfigEntry = ConfigEntry[VeSyncDataCoordinator]
```

Comment:
If you define this one, you should always use it everywhere in the integration.

For example, the init module doesn't use type.

Reply:
Removed.